backend/src/components/PublicEvents/model.ts

```diff
@@ -0,0 +1,66 @@
+import { Document, Schema, Types } from "mongoose";
+import mainDBConnection from "../../config/connection/main-db";
+
+export interface IPublicEventsModel extends Document {
+	_id: Types.ObjectId;
+	title: string;
+	description: string;
+	isActive: boolean;
+	created: Date;
+	startDate: Date;
+	endDate: Date;
+	maxScore: number;
+	tasks: Types.ObjectId[];
```

Comment:
you don't need to show tasks, score, isActive, users, participations on public api

Reply:
done